lang/en/zoom.php

```diff
@@ -108,7 +108,7 @@
 $string['on'] = 'On';
 $string['option_audio'] = 'Audio options';
 $string['option_host_video'] = 'Host video';
-$string['registration_type'] = 'Registration';
+$string['enable_regisration'] = 'Registration';
```

Comment:
@Ash-5 all is well except for the spelling of `registration` here. I know you would want to kill me. But trust me, spelling matters !